packages/mui-material/src/PigmentGrid/PigmentGrid.tsx

```diff
@@ -57,24 +57,29 @@ export interface GridBaseProps {
   offset?: ResponsiveStyleValue<number> | undefined;
   /**
    * @internal
-   * The level of the grid starts from `0`
-   * and increases when the grid nests inside another grid regardless of container or item.
+   * The level of the grid starts from `0` and increases when the grid nests
+   * inside another grid. Nesting is defined as a container Grid being a direct
+   * child of a container Grid.
    *
    * ```js
-   * <Grid> // level 0
-   *   <Grid> // level 1
-   *     <Grid> // level 2
-   *   <Grid> // level 1
+   * <Grid container> // level 0
+   *   <Grid container> // level 1
+   *     <Grid container> // level 2
    * ```
    *
-   * Only consecutive grid is considered nesting.
-   * A grid container will start at `0` if there are non-Grid element above it.
+   * Only consecutive grid is considered nesting. A grid container will start at
+   * `0` if there are non-Grid container element above it.
    *
    * ```js
-   * <Grid> // level 0
+   * <Grid container> // level 0
    *   <div>
-   *     <Grid> // level 0
-   *       <Grid> // level 1
+   *     <Grid container> // level 0
+   * ```
+   *
+   * ```js
+   * <Grid container> // level 0
+   *   <Grid>
+   *     <Grid container> // level 0
    * ```
    */
   unstable_level?: number;
```

Comment:
The PigmentGrid doesn't have the `unstable_level` prop ([reference](https://github.com/mui/pigment-css/blob/master/packages/pigment-css-react/src/Grid.jsx)), it shouldn't have been added. May I ask you to remove it? (or if you prefer, I can do it in a separate PR, let me know)

Reply:
Done